src/Tests/FunctionalTests/CCW/CCW.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>

```

Comment:
Should we make one of the functional tests use the newer one?

Reply:
Switched the collections test to the new analyzer 👍